tests/app/main/test_suppliers.py

```diff
@@ -799,6 +798,27 @@ def test_handles_supplier_with_no_registration_country_key(self, data_api_client
             res = self.client.get("/suppliers/details")
             assert res.status_code == 200
 
+    def test_shows_address_details(self, data_api_client):
+        data_api_client.get_supplier.return_value = get_supplier(registrationCountry="country:GB")
+
+        with self.app.test_client():
+            self.login()
+
+            response = self.client.get("/suppliers/details")
+            assert response.status_code == 200
+            page_html = response.get_data(as_text=True)
+            document = html.fromstring(page_html)
+
+            address1 = document.xpath("//span[text()='Registered company address']/following::td[1]/span/p/span[1]")
+            town = document.xpath("//span[text()='Registered company address']/following::td[1]/span/p/span[2]")
+            country = document.xpath("//span[text()='Registered company address']/following::td[1]/span/p/span[3]")
+            postcode = document.xpath("//span[text()='Registered company address']/following::td[1]/span/p/span[4]")
+
+            assert "1 Street" in address1[0].text
+            assert "Supplierville" in town[0].text
+            assert "United Kingdom" in country[0].text
+            assert "11 AB" in postcode[0].text
```

Comment:
I think the country should be after the postcode in the address - we'll need to update the frontend toolkit pattern to fix this.

Reply:
I'm on it. Here is a frontend-toolkit pull request: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/407

I will proceed to update test and toolkit version in supplier-frontend now :).